pkg/cluster/spec/tikv_cdc.go

```diff
@@ -0,0 +1,294 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spec
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/pingcap/tiup/pkg/cluster/api"
+	"github.com/pingcap/tiup/pkg/cluster/ctxt"
+	"github.com/pingcap/tiup/pkg/cluster/template/scripts"
+	logprinter "github.com/pingcap/tiup/pkg/logger/printer"
+	"github.com/pingcap/tiup/pkg/meta"
+)
+
+// TiKVCDCSpec represents the TiKVCDC topology specification in topology.yaml
+type TiKVCDCSpec struct {
+	Host            string                 `yaml:"host"`
+	SSHPort         int                    `yaml:"ssh_port,omitempty" validate:"ssh_port:editable"`
+	Imported        bool                   `yaml:"imported,omitempty"`
+	Patched         bool                   `yaml:"patched,omitempty"`
+	IgnoreExporter  bool                   `yaml:"ignore_exporter,omitempty"`
+	Port            int                    `yaml:"port" default:"8600"`
```

Comment:
The port should also be added to `portTypes` list in `pkg/cluster/spec/validate.go` at around L619, other wise the port conflict check is not working for it.

Reply:
I found that the `Port` is already in the list for [`portTypes`](https://github.com/pingcap/tiup/blob/3671d82636e29dd71f86ba6fb3cf6eaec141dda9/pkg/cluster/spec/validate.go#L620) where `portConflictsDetect` get port value from. So the port conflict check should work.

And I add a test case to verify port conflict check in integrated test for tikv-cdc. PTAL~